homeassistant/components/arve/config_flow.py

```diff
@@ -0,0 +1,68 @@
+"""Config flow for Arve integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from asyncarve import Arve, ArveConnectionError, ArveSensPro
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigFlowResult
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_CLIENT_SECRET, CONF_NAME
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ArveConfigFlowHadler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Arve."""
+
+    async def _show_setup_form(self, errors: dict[str, str] | None = None):
+        """Show the setup form to the user."""
+        return self.async_show_form(
+            step_id="user",
+            data_schema=vol.Schema(
+                {
+                    vol.Required(CONF_ACCESS_TOKEN): str,
+                    vol.Required(CONF_CLIENT_SECRET): str,
+                    vol.Required(CONF_NAME): str,
+                }
+            ),
+            errors=errors or {},
+        )
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle the initial step."""
+
+        if user_input is None:
+            return await self._show_setup_form(user_input)
+
+        self._async_abort_entries_match(user_input)
+
+        errors = {}
+
+        access_token = user_input[CONF_ACCESS_TOKEN]
+        customer_token = user_input[CONF_CLIENT_SECRET]
+        name = user_input[CONF_NAME]
+
+        arve = Arve(
+            access_token,
+            customer_token,
+            name,
+        )
```

Comment:
```suggestion
        arve = Arve(
              user_input[CONF_ACCESS_TOKEN],
              user_input[CONF_CLIENT_SECRET],
              user_input[CONF_NAME],
        )
```
didn't notice your parameters are named differently

you could also try
```python
        arve = Arve(
              *user_input.values()
        )
```

Reply:
I'd advice against your last suggestion of the `*user_input.values()`

Reply:
Fair enough, was also not quite sure if that'd really be a good idea